src/palm-react-native-ui-kit/components/atoms/Container.tsx

```diff
@@ -44,6 +40,48 @@ const Container = ({
       ) : (
         <View style={style}>{children}</View>
       )}
+    </>
+  )
+}
+
+const Container = ({
+  children,
+  style,
+  safeAreaBackgroundColor,
+  keyboardAvoiding,
+  scrollable,
+  disableSafeArea,
+}: {
+  children: ReactNode
+  style?: StyleProp<ViewStyle>
+  safeAreaBackgroundColor?: ColorValue
+  keyboardAvoiding?: boolean
+  scrollable?: boolean
+  disableSafeArea?: boolean
```

Comment:
change to `safeArea?` and if true, then wrap with safe area view

Reply:
changed.